src/styles/index.scss

```diff
@@ -1,3 +1,137 @@
+@import "./utils/variables";
+@import "./utils/mixin";
+
 body {
   margin: 0;
+  font-family: $general-font-family;
+  font-style: normal;
+}
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  background: $card-bg;
+  padding: 32px 16px 16px;
+
+
+
+  &__picture {
+    width: 160px;
+    height: 134px;
+    padding-left: 3px;
+    padding-bottom: 36px;
+  }
+
+  &__title {
+    margin: 0;
+    color: $card-title-text;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    width: 166px;
+  }
+
+  &__description {
+    margin: 0;
+    color: $card-description-text;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    margin-top: 4px;
+  }
+
+  &__reviews {
+    margin: 0;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    text-align: right;
+    color: $card-reviews-text;
+  }
+
+  &__prices {
+    @include flex-space-between;
+
+    margin-top: 24px;
+  }
+
+  &__text {
+    margin: 0;
+    color: $card-primary-text;
+    font-size: 12px;
+    font-weight: 400;
+    line-height: 18px;
+  }
+
+  &__price {
+    margin: 0;
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 18px;
+    color: $card-price-text;
+  }
+
+  &__button {
+    box-sizing: border-box;
+    margin: 0;
+
+    @include flex-center;
+
+    border: none;
+    width: 166px;
+    height: 40px;
```

Comment:
it's not a good practice to hardcode height for elements with content inside. use padding instead